### ISSUE TITLE
Example performance at higher resolutions

### ISSUE THREAD

**Issue by jdswebb** [GameFoundry/bsf#163]
When I increase the resolution of the examples (tested on Physics and SkeletalAnimation) I go from a steady 60fps at the default resolution, to an unstable 20-30fps at 1440p. This doesn't seem right for such simple scenes.

I'm using a 1080 GTX - curious if you can repro this or if it's some oddity with my machine.

**Comment by BearishSun** [comment]
Which platform are you running on, and which render API backend are you using? (DirectX is default on Windows, OpenGL on Linux/macOS if you haven't changed it). This is 1440p in windowed or fullscreen mode? (should be windowed unless you changed the examples)

I don't have a 1440p monitor to test this properly but I'll see if I can notice a performance drop with higher resolutions once I know the info above.

**Comment by jdswebb** [comment]
Yeah sorry this is on Windows with everything else unchanged - so windowed and DirectX.

It gets choppy for me even at 1080p - FPS shows as about 50 but it's not smooth.

**Comment by BearishSun** [comment]
It stays well above 60 for me at 1080p, but I do notice an unreasonable FPS drop with resolution increase. I'll let you know when I track it down.

**Comment by athanase** [comment]
Hi there,
I do have a 1440p screen. And no performance issues whatsoever. 
I am running under Linux with the nvidia drivers version 396.24.

**Comment by BearishSun** [comment]
I did make some performance improvements since this thread was opened, and I'm still looking into a few potential problem areas.
